src/Evolution/DgSubcell/Tags/Jacobians.hpp

```diff
@@ -39,6 +39,13 @@ struct DetInverseJacobianLogicalToGrid : db::SimpleTag {
   using type = Scalar<DataVector>;
 };
 
+/// \brief The determinant of the inverse Jacobian from the element logical
+/// frame to the inertial frame at the cell centers.
+struct DetInverseJacobianLogicalToInertial : db::SimpleTag {
```

Comment:
Want to use this in `Evolution/DgSubcell/Tags/ObserverCoordinates.hpp` (in `ObserverJacobianAndDetInvJacobian`)? (Side note: those Jacobian compute tags probably need a `Compute` suffix and should move to their own file)

Reply:
Those tags also support `Grid->Inertial` maps, which isn't yet generalized. Is it okay if I leave that for now?

Reply:
Sure